### PR TITLE
FTWD Miami Osse Laborde

### PR DESCRIPTION
**Pull request by olaborde**

### CODE REVIEW

File: starter-code/css/main.css [hunk 1]
```diff
@@ -8,3 +8,91 @@ body {
...
+  /* text-align: right; */
+  
+  font-family: Billabong;
+ 
+}
+
+
+
+.logo span {
+  position: relative;
```
Comment (MarcosPalaciosIII) [inline starter-code/css/main.css]: logo needs to be moved down more. review what position relative does.

File: starter-code/css/main.css [hunk 2]
```diff
@@ -8,3 +8,91 @@ body {
...
+  font-size: 20px;
+}
+
+.logo2  {
+  background-color: #4881ab;
+  position: absolute;
+  top:45px;
+  right: 300px;  
+  margin-left: 900px;
+  font-size: 20px;
```
Comment (MarcosPalaciosIII) [inline starter-code/css/main.css]: review what position: absolute does. This should also be moved down more.

File: starter-code/css/main.css [hunk 3]
```diff
@@ -8,3 +8,91 @@ body {
...
+  
+
+
+.footer{
+
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 1rem;
```
Comment (MarcosPalaciosIII) [inline starter-code/css/main.css]: again review what position: absolute does. the footer should be moved up some more.

File: starter-code/css/main.css [hunk 4]
```diff
@@ -8,3 +8,91 @@ body {
...
+    background-color: rgb(255, 255, 255);
+    width: 300px;
+    border: 25px solid white;
+    border-radius: 12px;
+    padding: 25px;
+    margin-left: 605px;
+    margin-bottom: 00px;
+    position: fixed;
+    left: 60px;
+    bottom: 150px;
```
Comment (MarcosPalaciosIII) [inline starter-code/css/main.css]: fixed position will have the content moving with the screen. so if looking from smaller screen it moves to a location that you may not want it to be at.

**Comment by MarcosPalaciosIII** [comment]
need to have content moved in more and try not to use the position: feature so much, especially when working with responsive design.  Not bad just needs a bit of improvement.

**Comment by olaborde** [comment]
I’ll work on it more

Sent from my iPhone

On Apr 6, 2018, at 10:09 AM, MarcosPalaciosIII <notifications@github.com<mailto:notifications@github.com>> wrote:


@MarcosPalaciosIII commented on this pull request.

________________________________

In starter-code/css/main.css<https://github.com/ironhack-labs/lab-css-instagram-clone/pull/438#discussion_r179755788>:

> +
+
+.logo{
+
+  background-color: #4881ab;
+  height: 200px;
+  /* text-align: right; */
+
+  font-family: Billabong;
+
+}
+
+
+
+.logo span {
+  position: relative;


logo needs to be moved down more. review what position relative does.

________________________________

In starter-code/css/main.css<https://github.com/ironhack-labs/lab-css-instagram-clone/pull/438#discussion_r179756101>:

> +
+.logo span {
+  position: relative;
+  top:20px;
+  right: 240px;
+  margin-left: 900px;
+  font-size: 20px;
+}
+
+.logo2  {
+  background-color: #4881ab;
+  position: absolute;
+  top:45px;
+  right: 300px;
+  margin-left: 900px;
+  font-size: 20px;


review what position: absolute does. This should also be moved down more.

________________________________

In starter-code/css/main.css<https://github.com/ironhack-labs/lab-css-instagram-clone/pull/438#discussion_r179756286>:

> +  background-color: #4881ab;
+
+}
+
+
+
+
+
+
+.footer{
+
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 1rem;


again review what position: absolute does. the footer should be moved up some more.

________________________________

In starter-code/css/main.css<https://github.com/ironhack-labs/lab-css-instagram-clone/pull/438#discussion_r179767119>:

> +
+}
+
+.content{
+    float: right;
+    background-image: url("images/frames");
+    background-color: rgb(255, 255, 255);
+    width: 300px;
+    border: 25px solid white;
+    border-radius: 12px;
+    padding: 25px;
+    margin-left: 605px;
+    margin-bottom: 00px;
+    position: fixed;
+    left: 60px;
+    bottom: 150px;


fixed position will have the content moving with the screen. so if looking from smaller screen it moves to a location that you may not want it to be at.

**Pull request closed without merging**